string frame_id
---
string frame_id
bool connected
bool armed
string mode
float32 x
float32 y
float32 z
float64 lat
float64 lon
float32 alt
float32 vx
float32 vy
float32 vz
float32 roll
float32 pitch
float32 yaw
float32 roll_rate
float32 pitch_rate
float32 yaw_rate
float32 voltage
float32 cell_voltage
